projects/alfresco/asg/main.tf

```diff
@@ -129,8 +129,11 @@ data "template_file" "user_data" {
     bucket_encrypt_type = "kms"
     bucket_key_id       = "${var.bucket_kms_key_id}"
     external_fqdn       = "${var.app_hostnames["external"]}.${var.external_domain}"
+<<<<<<< HEAD
+=======
     
     # For bootstrapping
+>>>>>>> 6059dc65328dc79a5ccd83c2f66ab7d018fcef48
```

Comment:
should this be here? same for line 132>132

Reply:
removed tags